RocketChat/Rocket.Chat.ReactNative
# [FIX] RoomInfoView displaying different info depending on the origin

## Proposed changes


Navigating to RoomInfoView from Avatar and from RoomActionsView displays different information.

## Issue(s)	


## How to test or reproduce

- Make sure you're logged in user with admin (or correct permission)
- Go to a room, and click on the avatar of another user that talked before
- You will be moved directly to RoomInfoView and you can see the user's roles
- Then, create a DM to the same user that you clicked before,
- Click on Header, then when you are inside the Actions, click on the user's name
- You will see the RoomInfoView and should appear the roles too

## Screenshots

### Before

https://user-images.githubusercontent.com/47038980/148281211-f9de2bc0-b289-486c-b3d0-b9a4aa994602.mov

### After

https://user-images.githubusercontent.com/47038980/148281336-8b4e6bb1-89d9-49a5-b2b7-ec197926cf6a.mov




## Types of changes




## Checklist



## Further comments
